### DISCUSSION TITLE
.chezmoiignore does not work

### DISCUSSION THREAD

**Discussion by ghost** [twpayne/chezmoi#2851] [General]
Hello
I have in .local/share/chezmoi/.chezmoiignore
weechat/certs/*
weechat/ssl/*
when I do chezmoi add .config/weechat
those folder are added, while I don't want them to be added

**Comment by triallax** [comment]
Have you tried
```
.config/weechat/certs/
.config/weechat/ssl/
```
instead? I think your patterns do not apply to subdirectories of the target path, but I'm not sure.

**Reply by halostatue** [reply]
The [reference on chezmoiignore](https://www.chezmoi.io/reference/special-files-and-directories/chezmoiignore/) appears to confirm this. You could also add `.chezmoiignore` to `dot_config/weechat` and have it ignore `certs/` and `ssl/`.

**Comment by ghost** [comment]
Thank you it worked
